neurobooth_terra/dataflow.py

```diff
@@ -349,7 +347,7 @@ def delete_files(db_table, target_dir, suitable_dest_dir1, suitable_dest_dir2,
     been copied to the *final* destination "who", we provide the following
     arguments to the function:
 
-    target_dir = NAS, suitable_dest_dir1 = who, suitable_dest_dir2 = neo
+    target_dir = NAS, suitable_dest_dir = [who, neo]
```

Comment:
i may have missed it, but if you aren't verifying that there are two (at least?) values in suitable_dest_dir that might be a good idea

Reply:
The list needs at least a single value for the function to work, and can have any number of values in it. I added a length check which raises a ValueError if not met. Thanks!